gufe/tokenization.py

```diff
@@ -54,7 +54,9 @@ def __new__(cls, name, bases, classdict):
     def __call__(cls, *args, **kwargs):
         instance = super().__call__(*args, **kwargs)
         # add to registry if not already present
-        TOKENIZABLE_REGISTRY.setdefault(instance.key, instance)
+        key = instance.key
```

Comment:
You were not kidding when you said that it would take 2 lines of code to change this :rofl: